aggregator/src/aggregator/error.rs

```diff
@@ -63,15 +63,18 @@ pub enum Error {
     /// An attempt was made to act on an unknown aggregation job.
     #[error("task {0}: unrecognized aggregation job: {1}")]
     UnrecognizedAggregationJob(TaskId, AggregationJobId),
+    /// An attempt was made to act on a known but deleted aggregation job.
+    #[error("deleted aggregation job: {0}")]
+    DeletedAggregationJob(TaskId, AggregationJobId),
     /// An attempt was made to act on an unknown collection job.
     #[error("unrecognized collection job: {0}")]
-    UnrecognizedCollectionJob(CollectionJobId),
+    UnrecognizedCollectionJob(TaskId, CollectionJobId),
```

Comment:
Noting that the `AggregationJob` error variants include `TaskId`, I opted to augment the `CollectionJob` errors to match.